coop_print_badge/views/menu.xml

```diff
@@ -14,7 +14,7 @@
         <field name="action" ref="action_badge_to_print" />
         <field name="parent_id" ref="coop_shift.menu_reporting_shifts" />
         <field name="sequence">20</field>
-        <field name="groups_id" eval="[(6, 0, [ref('coop_print_badge.group_access_badge_report')])]" />
+        <field name="groups_id" eval="[(6, 0, [ref('coop_shift.group_shift_manager')])]" />
```

Comment:
I think we said the membership user would have access to this, no? :thinking: 
It shouldn't require the manager group

Reply:
In fact you can remove this line I believe, because this menu is a child of the members menu, so it should be ok. Correct me if I'm wrong

Reply:
:thinking: Okay, will add the membership user.